package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yoti",
-  "version": "3.8.0",
+  "version": "3.8.1",
```

Comment:
Not related to this PR but going forwards we're updating version numbers to the candidate number after a release, not before it.

Reply:
I was originally think we should do this for the benefit of sonar scanning. I don't think it's as important to update it here after each release, but I guess we might as well if we're updating it for sonar.